From: Aram Akhavan <github@aram.nubmail.ca>
Date: Sat, 15 Jul 2023 13:16:58 -0400
Subject: nfs-idmapd.service: add network-online.target to Wants= and After=
Origin: https://git.linux-nfs.org/?p=steved/nfs-utils.git;a=commit;h=4f5e66718c01f89b28bf9e7a8c645d9e18682485
Bug-Debian: https://bugs.debian.org/1035840

nfs-idmapd.service does not have any dependency on the network so there's no
starting point to wait for DNS resolution. nfs-server.service already has this
network dependency and ordering.

Signed-off-by: Aram Akhavan <github@aram.nubmail.ca>
Signed-off-by: Steve Dickson <steved@redhat.com>
---
 systemd/nfs-idmapd.service | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/systemd/nfs-idmapd.service b/systemd/nfs-idmapd.service
index f38fe527237e..198ca87c2bc9 100644
--- a/systemd/nfs-idmapd.service
+++ b/systemd/nfs-idmapd.service
@@ -2,7 +2,8 @@
 Description=NFSv4 ID-name mapping service
 DefaultDependencies=no
 Requires=rpc_pipefs.target
-After=rpc_pipefs.target local-fs.target
+After=rpc_pipefs.target local-fs.target network-online.target
+Wants=network-online.target
 
 BindsTo=nfs-server.service
 
-- 
2.40.1

